app-forensics/scalpel/metadata.xml

```diff
@@ -1,15 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <!-- maintainer-needed -->
-  <longdescription lang="en">
-    Scalpel is a fast file carver that reads a database of header and footer
-    definitions and extracts matching files or data fragments from a set of image
-    files or raw device files. Scalpel is filesystem-independent and will carve
-    files from FATx, NTFS, ext2/3, HFS+, or raw partitions. It is useful for both
-    digital forensics investigation and file recovery.
-  </longdescription>
-  <upstream>
-    <remote-id type="github">sleuthkit/scalpel</remote-id>
-  </upstream>
+	<maintainer type="person" proxied="yes">
+		<email>martin.dummer@gmx.net</email>
+		<name>Martin Dummer</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription lang="en">
```

Comment:
The diff on this could be reduced by keeping the current indentation style. No strong opinion on this, though.